components/binding-usage-controller/docs/proposals/usage-kind.md

```diff
@@ -0,0 +1,94 @@
+# Binding Usage Target proposal
+
+## Overview
+The **UsageKind** allows dynamic (in the runtime) **Binding Usage Controller** configuration. It allows to define, what (and how) resources can be bound with **ServiceBinding**. Currently, only two resources can be bound: **Deployment** and **Function**. Adding a new resource requires changes in the Binding Usage Controller code. The goal is to make it possible only with configuration.
+
+## UsageKind 
+
+The idea is to have a new cluster wide custom resource, which defines bindable resource, for example
+```yaml
+apiVersion: servicecatalog.kyma.cx/v1alpha1
+kind: UsageKind
+metadata:
+   name: function
+spec:
+   # displayName is a human readable name of the target
+   displayName: Function
+   
+   # the type of resource (target), which is labelled by the controller, is specified as resource group, kind and version (all fields required)
```

Comment:
`# the type of the resource and the target, which is labeled by the controller, is specified as a resource group, kind, and version, with all fields required`

Reply:
done